src/flexbox.rs

```diff
@@ -300,16 +303,27 @@ impl Forest {
     #[inline]
     #[must_use]
     fn determine_available_space(
-        node_size: Size<Number>,
-        parent_size: Size<Number>,
+        node_size: Size<Option<f32>>,
+        parent_size: Size<Option<f32>>,
         constants: &AlgoConstants,
-    ) -> Size<Number> {
-        Size {
-            width: node_size.width.or_else(parent_size.width - constants.margin.horizontal_axis_sum())
-                - constants.padding_border.horizontal_axis_sum(),
```

Comment:
The original implementation here was wrong; padding should only be subtracted if the node width is undefined.

Reply:
None of the tests seem to care either way about this change though 🙃